src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteTypeVisotor.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+using Microsoft.OpenApi.Models;
+
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Visitors
+{
+    /// <summary>
+    /// This represents the type visitor for <see cref="byte"/>.
+    /// </summary>
+    public class ByteTypeVisitor : TypeVisitor
```

Comment:
According to the [OpenAPI v3 spec - data types](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md#data-types), we need byte support - string data type with base64 format. It's basically byte array (`byte[]`). Therefore, we need `ByteArrayTypeVisitor`, instead of `ByteTypeVisitor` and it only handles the byte array value.

Reply:
Also OpenAPI.NET expects `byte[]` not `byte`: https://github.com/microsoft/OpenAPI.NET/blob/vnext/src/Microsoft.OpenApi/Any/OpenApiBinary.cs

Could you please take a further look?